node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -83,71 +93,123 @@ func NewMessageService(ip string, port int, me types.Address, pk []byte, useMdns
 	options := []libp2p.Option{
 		libp2p.Identity(messageKey),
 		libp2p.ListenAddrStrings(fmt.Sprintf("/ip4/%s/tcp/%d", ip, port)),
-		libp2p.Transport(tcp.NewTCPTransport),
-		libp2p.NoSecurity,
-		libp2p.DefaultMuxers,
+		// libp2p.Transport(tcp.NewTCPTransport),
```

Comment:
Should these still be commented out?

Reply:
I forgot I had done that. I just added them back in